src/main/java/stream/flarebot/flarebot/util/pagination/PagedEmbedBuilder.java

```diff
@@ -65,13 +68,34 @@ public PagedEmbed build() {
         boolean pageCounts = false;
         if (list.getPages() > 1)
             pageCounts = true;
-        return new PagedEmbed(title, codeBlock, hasCodeBlock, list, pageCounts, color);
+        return new PagedEmbed(title, codeBlock, hasCodeBlock, list, pageCounts, color, groupPrefix, groupsPerPage);
     }
 
     public void setColor(Color color) {
         this.color = color;
     }
 
+    /**
+     * Set the embed to use the groups within the {@link PaginationList}.
+     */
+    public void useGroups() {
+        this.groupPrefix = "";
+    }
+
+    public void useGroups(int groupsPerPage) {
+        this.groupPrefix = "";
```

Comment:
Why do you even set it to an empty string?

Reply:
because right now it's null. I didn't want to have a boolean for if groups are enabled when we can have this variable do that